### PR TITLE
Use logarithmic depth buffer

### PR DESCRIPTION
**Pull request by martinRenou**
This prevents visual artifacts with big/small objects

### CODE REVIEW

**Comment by arjxn-py** [comment]
Bot please update snapshots

File: ui-tests/tests/ui.spec.ts-snapshots/JCAD-Undo-linux.png
Comment (martinRenou) [inline ui-tests/tests/ui.spec.ts-snapshots/JCAD-Undo-linux.png:1]: 👀 wut

**Review by arjxn-py — APPROVED** [review]
Works well, thanks @martinRenou 

The failure looks unrelated, maybe you want to re-trigger that one?

**Comment by martinRenou** [comment]
The visual regression makes me nervous though, I don't understand why the color of the box changes in that case

**Comment by arjxn-py** [comment]
> The visual regression makes me nervous though, I don't understand why the color of the box changes in that case

It must be since the `SELECTED_MESH_COLOR` is removed, not sure why it's no longer selected with bounding box and why it's happening in this PR in the first place

**Comment by martinRenou** [comment]
Ok well let's go and merge! Thanks

**Pull request merged**